packages/devtools_app/lib/src/debugger/flutter/console.dart

```diff
@@ -70,7 +70,9 @@ class _ConsoleState extends State<Console> {
                     controller: scrollController,
                     itemBuilder: (context, index) {
                       return RichText(
-                        text: TextSpan(children: maybeConvertToAnsiText(lines[index], textStyle)),
+                        text: TextSpan(
+                            children: maybeConvertToAnsiText(
+                                lines[index], textStyle)),
```

Comment:
general nit: add trailing commas to make the dartfmt use flutter style pretty much any time you get more than one comma on the same line.

Reply:
In this case should it be a comma after `textStyle` and a comma after the first parens? And then flutter style puts every item ending in a comma in its own line?

Reply:
in this case I would add a comma before both parens. That way you get the flutter style with indents of only 2 spaces which makes deeply nested flutter build methods easier to read.